SearchManager.h

```diff
@@ -25,8 +20,9 @@ class SearchManager {
 		}
 	}
 
-	PlayerInfo get_player_info() {
-		return *active_searches.begin();
+	PlayerInfo get_player() {
```

Comment:
This name implies that the class has a field of `Player` or some subclass of Player. Change it.